internal/controller/management_controller.go

```diff
@@ -252,6 +258,15 @@ func (r *ManagementReconciler) enableAdmissionWebhook(ctx context.Context, mgmt
 
 	admissionWebhookValues["enabled"] = true
 	config["admissionWebhook"] = admissionWebhookValues
+
+	// Enable HMC capi operator only if it was not explicitly disabled in the config to
+	// support installation with existing cluster api operator
+	if capiOperatorValues["enabled"] != false {
```

Comment:
This looks a bit weird to me. I think if we want to preserve the current API (keep CAPI as a separate component of the Management object), we should make the CAPI field optional (now it's required in the CRD)

Reply:
@Kshatrix I'm not sure I fully understood. This condition only affects the `CAPI Operator` subchart, not the CAPI itself. To support an external installation of the CAPI operator (in case it already exists on the management cluster). However, CAPI remains a separate component of the management object (no changes here). Could you please elaborate?

Reply:
Nvm, I overlooked that operator is installed as part of HMC itself. If it requires certmanager to be running, it makes sense. Thanks!